src/lang/lwc_cn.properties

```diff
@@ -22,23 +22,31 @@ you=你
 chest=箱子
 furnace=熔炉
 dispenser=发射器
+dropper=Dropper
 sign=木牌
-wooden_door=橡木门
```

Comment:
You shouldn't be deleting valid translations, although for this case you can probably safely rename this to oak_door.